pkg/test/helpers.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2023 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
```

Comment:
Can `test_helper` be a better name ?

Reply:
throwing my preference into the ring. I prefer naming the package after the directory it is in.

Reply:
I agree we should make the package name consistent with the directory name and I assume we rename the directory when we rename the package.